vue-app/src/components/WalletWidget.vue

```diff
@@ -55,6 +59,8 @@ export default class WalletWidget extends Vue {
   // Boolean to only show Connect button, styled like an action button,
   // which hides the widget that would otherwise display after connecting
   @Prop() isActionButton!: boolean
+  // Boolean to allow connect button to be full width
+  @Prop() fullWidth!: boolean
```

Comment:
Can we rename this to `fullWidthMobile` to make that clear?

Reply:
Agreed since the styles only apply if the user is on mobile.